package.json

```diff
@@ -11,21 +11,20 @@
     "dependencies": {
         "artedi": "1.1.0",
         "assert-plus": "1.0.0",
-        "bignum": "0.6.0",
+        "bignum": "0.13.0",
         "bunyan": "0.22.1",
         "clone": "0.1.9",
-        "dtrace-provider": "0.2.8",
-        "fash": "2.5.0",
+        "dtrace-provider": "0.8.8",
+        "fash": "git+https://github.com/joyent/node-fash.git#MANTA-4720",
         "fast": "2.7.0",
         "jsprim": "2.0.0",
         "kang": "1.2.0",
```

Comment:
from the kang CHANGES.md:

```
## v1.3.0

* #7 Update restify dep, drop dep on restify-clients. An important
  part of this is getting deps with a more modern dtrace-provider
  for support building against newer versions of Node.
```

let's update to 1.3.0.

Reply:
will do